aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -224,6 +224,28 @@ Routes can be [CORS](xref:security/cors?view=aspnetcore-6.0) enabled using [CORS
 
 For more information, see <xref:security/cors?view=aspnetcore-6.0>
 
+## ValidateScopes and ValidateOnBuild
+
+<xref:Microsoft.Extensions.DependencyInjection.ServiceProviderOptions.ValidateScopes> and <xref:Microsoft.Extensions.DependencyInjection.ServiceProviderOptions.ValidateOnBuild> are enabled by default in the [Development](xref:fundamentals/environments) but disabled in release mode.
+
+When `ValidateOnBuild=true`, the DI container validates the service configuration at build time. If the service configuration is invalid, the build fails at app startup, rather than at runtime when the service is requested.
```

Comment:
```suggestion
When `ValidateOnBuild` is `true`, the DI container validates the service configuration at build time. If the service configuration is invalid, the build fails at app startup, rather than at runtime when the service is requested.
```

Reply:
Also, I think "at build time" and "the build fails" are a bit confusing. I realize it means "when the container is built", but it could easily be understood as when you build (compile) the application. Ideally it should be reworded to remove the ambiguity.